src/main/java/nl/fuchsia/controller/PersoonController.java

```diff
@@ -37,9 +37,13 @@ public PersoonController(PersoonService persoonService) {
      * @param persoon - De toe te voegen persoons gegevens.
      * @return - Geeft de toegevoegde persoon terug in het formaat van een Json object.
      */
-    @PostMapping(path = "/add")
+    @PostMapping
     public ResponseEntity<Persoon> addPersoon(@Valid @RequestBody Persoon persoon) {
-        persoonService.addPersoon(persoon);
-        return ResponseEntity.ok(persoon);
+        return ResponseEntity.ok(persoonService.addPersoon(persoon));
     }
-}
+
```

Comment:
Hier misschien wel comment bij zit in andere US.

Reply:
Gaat om methode persoonById. Zit inderdaad nergens in een story. We laten deze staan.